src/cmd/init.rs

```diff
@@ -45,13 +58,20 @@ pub fn execute(args: &ArgMatches) -> Result<()> {
         }
     }
 
-    println!("\nDo you want a .gitignore to be created? (y/n)");
-
-    if confirm() {
+    if args.is_present("gitignore") {
```

Comment:
There doesn't appear to be a way to pass "no" on the command-line for gitignore.  Perhaps the flag should take a value?  Maybe something like `--ignore=git` or `--ignore=none`?

Reply:
The arg is now `--ingore` and it either takes the value `git` (creates a .gitignore), `none` doesn't create a .gitignore, or if neither is present, the user is prompted.